benchmark/scenarios/rate_measurer.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2015 ClusterHQ Inc.  See LICENSE file for details.
```

Comment:
Rename this file with an underscore prefix, as it should not be exported from this module.